src/ucp/tag/rndv.c

```diff
@@ -226,6 +227,49 @@ ucs_status_t ucp_tag_send_start_rndv(ucp_request_t *sreq)
     return status;
 }
 
+static UCS_F_ALWAYS_INLINE size_t
+ucp_rndv_adjust_zcopy_length(size_t min_zcopy, size_t max_zcopy, size_t align,
```

Comment:
maybe rename to ucp_rndv_adjust_**get**_zcopy_length?

Reply:
we are using this function for pipeline PUT Zcopy

Reply:
seems it's used for ucp_rndv_send_frag_**get**_mem_type

Reply:
this function is used for:
- `ucp_rndv_progress_rma_get_zcopy`
- `ucp_rndv_send_start_put_pipeline`
- `ucp_rndv_recv_start_get_pipeline`